src/Convert/ConvertInterface.php

```diff
@@ -148,6 +162,15 @@ public function toDegreesMinutesSeconds($format = ConvertInterface::DEFAULT_DMS_
      */
     public function toDecimalMinutes($format = ConvertInterface::DEFAULT_DM_FORMAT);
 
+    /**
+     * Convert and format a decimal degree coordinate to degree decimal minutes coordinate.
+     *
+     * @param string $format The way to format the DDM coordinate.
+     *
+     * @return string Converted and formatted string.
+     */
+    public function toDegreeDecimalMinutes($format = ConvertInterface::DEFAULT_DDM_FORMAT);
```

Comment:
This is indeed a BC break. We cannot add new methods to an interface. We have to release 0.9 if we merge this.